tests/boardfarm_plugins/run_bf.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+scriptdir=$(dirname $(readlink -f ${0}))
+bf_plugin_dir=${scriptdir}
+bf_dir=$(realpath "${scriptdir}/../../../boardfarm/boardfarm")
+prplMesh_dir=$(realpath "${scriptdir}/..")
+
+if [ -n "${PYTHONPATH}" ]; then
+   PYTHONPATH="${bf_dir}:${bf_plugin_dir}:${prplMesh_dir}:${PYTHONPATH}"
+else
+   PYTHONPATH="${bf_dir}:${bf_plugin_dir}:${prplMesh_dir}"
+fi
+export PYTHONPATH
+
+exec bft -c boardfarm_sample_plugin/prplmesh_config.json -n prplmesh_dummy -x sample-suite
```

Comment:
Does i correctly understand that if you pass args (`-c`, `-n`, `-x`) to `bft` here then you are not planning that `run_bf.sh` will be used as Boardfarm run script?

Reply:
> Does i correctly understand that if you pass args (`-c`, `-n`, `-x`) to `bft` here then you are not planning that `run_bf.sh` will be used as Boardfarm run script?

Script's current version is supposed to be used only for demonstration of sample plugin.
When in #1071 there will be added operational testsuite where I'll provide a bit more advanced launcher script.